inmem/kv.go

```diff
@@ -284,10 +284,9 @@ func (b *Bucket) ForwardCursor(seek []byte, opts ...kv.CursorOption) (kv.Forward
 		defer close(pairs)
 
 		var (
-			batch     []pair
-			fn        = config.Hints.PredicateFn
-			iterate   = b.ascend
-			skipFirst = config.SkipFirst
+			batch   []pair
```

Comment:
Note for me. I did this because I thought the problem was in `inmem` but I found it wasn't. So perhaps I should put this back 🤔